clients/include/utility.hpp

```diff
@@ -2554,6 +2554,270 @@ void host_csrmm(J                     M,
     }
 }
 
+template <typename T, typename I, typename J>
+void host_csrmm_batched(J                    M,
+                        J                    N,
+                        J                    K,
+                        J                    batch_count_A,
+                        J                    offsets_batch_stride_A,
+                        I                    columns_values_batch_stride_A,
+                        hipsparseOperation_t transA,
+                        hipsparseOperation_t transB,
+                        T                    alpha,
+                        const I*             csr_row_ptr_A,
```

Comment:
__restrict__

Reply:
Given that none of the other routines in this file use restrict I think ill go through and fix all of them in a separate PR. That way I am not touching 100 different unrelated utility routines in this PR.